roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -33,6 +28,18 @@
   it { should be_owned_by 'aodh' }
 end
 
+{% if inventory_hostname in groups['controller'] %}
+processes = ['aodh-api',
+  'aodh-evaluator',
+  'aodh-listener',
+  'aodh-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'aodh' }
```

Comment:
yeah serverspec is not for checking that a process is running,  we have alerts for that.   serverspec should check that it's "enabled" maybe,  but not running.